src/kernel/code_page.zig

```diff
@@ -0,0 +1,88 @@
+const std = @import("std");
```

Comment:
Since this is more of a supporting file, it could be moved to src/kernel/code_page